application/factory.py

```diff
@@ -124,6 +126,7 @@ def static_subdir(subdir1, subdir2, file_name):
     app.after_request(harden_app)
 
     # Render jinja templates with less whitespace; applies to both CMS and static build
+    app.jinja_env.undefined = StrictUndefined
```

Comment:
The comment above relates to the settings below, might be nice to keep them together.

Reply:
Ohp, yep.